src/commands/connections/revealConnectionInAppSettings.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { CosmosDBConnection } from '../../explorer/CosmosDBConnection';
+import { ext } from "../../extensionVariables";
+
+export async function revealConnectionInAppSettings(node: CosmosDBConnection): Promise<void> {
+    const nodeToReveal = await ext.tree.findTreeItem(`${node.parent.parent.parent.appSettingsNode.fullId}/${node.appSettingKey}`);
+    if (nodeToReveal) {
```

Comment:
This should throw an error if it can't find the node, rather than silently failing. Something like `Failed to find app setting with key "...".`